languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/multiwords.txt

```diff
@@ -1,4 +1,52 @@
 #French multiwords file used for chunking
+au mieux A
+charge acquéreur N f s
+bas de contention N m sp
+satisfaction client N f s
+relation client N f s
+fiche client N f s
+fiches client N f p
+base de données client N f s
+bases de données client N f p
+un tantinet A
+latina J f sp
+langue maternelle N f s
+mauvais goût N s p
+tout de même A
+Troie Z f s
+a première vue A
```

Comment:
a → à

Reply:
One token words like `Troie` should be added to added.txt, not here.